docs/setup.asciidoc

```diff
@@ -40,22 +40,26 @@ You can skip this step if your `minSdkVersion` is 26 or higher.
 [[adding-gradle-plugin]]
 ==== Add the Elastic Agent Gradle plugin
 
-To automatically instrument <<supported-technologies,Supported Technologies>>, add the https://plugins.gradle.org/plugin/co.elastic.apm.android[Elastic APM agent plugin] to your application's `build.gradle` file as shown below:
+To automatically instrument <<supported-technologies,Supported Technologies>>, add the https://plugins.gradle.org/plugin/co.elastic.apm.android/0.15.0[Elastic APM agent plugin] to your application's `build.gradle` file as shown below:
 
 [source,groovy]
 ----
 // Android app's build.gradle file
 plugins {
     id "com.android.application"
-    id "co.elastic.apm.android" version "[latest_version]" <1>
+    id "co.elastic.apm.android" version "0.15.0"
 }
 ----
 
-<1> The Elastic plugin declaration must be added below the Android app plugin declaration (`com.android.application`) and below the Kotlin plugin declaration (if used).
-
 After adding the agent plugin, configure it.
 A minimal configuration sets the Elastic APM Server endpoint as shown below:
 
+NOTE: We unintentionally released a 1.x version, `1.13.1`.
+Please avoid using it as it doesn't contain our latest changes and we will continue to release `0.x` versions for now.
+Please make sure to only use `0.x.x` versions of our plugin until further notice.
+Please avoid using it as it doesn't contain our latest changes.
```

Comment:
These lines weren't replaced when you accepted the suggestion, so I think they need to be updated. Also, the `NOTE` below should probably match this one.

Reply:
Thanks, it's updated now.